SpaceShooterWithStats/Assets/Scripts/SpaceShooterTutorial/GameController.cs

```diff
@@ -38,6 +38,65 @@ private void SaveStatisticsToBrainCloud()
 		brainCloudStatusText.gameObject.SetActive(true);
 	}
 
+	void addScoreToLeaderBoard()
+	{
+		//sending to the cloud
+		App.Bc.SocialLeaderboardService.PostScoreToLeaderboard("Leaderboard", m_score, "{ \"" + "username" + "\" : \"" + PlayerPrefs.GetString("username") + "\"}");
+	}
+
+	void ReadLeaderBoard()
+	{
+		App.Bc.SocialLeaderboardService.GetGlobalLeaderboardView("Leaderboard", BrainCloud.BrainCloudSocialLeaderboard.SortOrder.HIGH_TO_LOW, 2, 2, LeaderBoardSuccess_Callback, StatsFailure_Callback);
+	}
+
+	void DisplayLeaderBoard(string data)
```

Comment:
Maybe call this Display top 5? kinda weird to see a for loop for 5... Maybe just leave a comment saying you're grabbing top 5? Hmm are you even taking top 5? I'm confused haha